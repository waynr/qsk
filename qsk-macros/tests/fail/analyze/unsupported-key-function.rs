use qsk_macros::layer;

fn main() {
    layer!(
        ModLayer[Meow]: {
            F -> MEOW(Navigation),
        },
    );
}
