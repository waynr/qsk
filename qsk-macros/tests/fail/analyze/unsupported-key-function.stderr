error: invalid key function

         = help: valid key functions include: ["TT", "TapToggle", "Exit"]

 --> tests/fail/analyze/unsupported-key-function.rs:6:18
  |
6 |             F -> MEOW(Navigation),
  |                  ^^^^
